test/unit/org/apache/cassandra/service/accord/CommandsForRangesTest.java

```diff
@@ -1,131 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package org.apache.cassandra.service.accord;
-
-import java.util.ArrayList;
-import java.util.Collections;
-import java.util.List;
-import java.util.TreeMap;
-
-import org.junit.Test;
-
-import accord.api.RoutingKey;
-import accord.impl.IntKey;
-import accord.primitives.SaveStatus;
-import accord.primitives.Range;
-import accord.primitives.Ranges;
-import accord.primitives.Timestamp;
-import accord.primitives.TxnId;
-import accord.utils.AccordGens;
-import accord.utils.Gen;
-import accord.utils.Gens;
-import org.apache.cassandra.service.accord.CommandsForRangesLoader.Summary;
-
-import static accord.utils.Property.qt;
-import static org.assertj.core.api.Assertions.assertThat;
-
-public class CommandsForRangesTest
```

Comment:
why was this test removed?

Reply:
this wasn't testing anything anymore, and I think wasn't even before this patch.